test/powershell/Modules/Microsoft.PowerShell.Utility/Import-Csv.Tests.ps1

```diff
@@ -22,6 +22,25 @@ Describe "Import-Csv DRT Unit Tests" -Tags "CI" {
     }
 }
 
+Describe "Import-Csv Double Quote Delimiter" -Tags "CI" {
+    BeforeAll {
+        $TestImportCsvQuoteDelimiter_EmptyValue = Join-Path -Path (Join-Path $PSScriptRoot -ChildPath assets) -ChildPath TestImportCsvQuoteDelimiter_EmptyValue.csv
+        $TestImportCsvQuoteDelimiter_QuoteWithValue = Join-Path -Path (Join-Path $PSScriptRoot -ChildPath assets) -ChildPath TestImportCsvQuoteDelimiter_QuoteWithValue.csv
+        $TestImportCsvQuoteDelimiter_QuoteCommaDelimiter = Join-Path -Path (Join-Path $PSScriptRoot -ChildPath assets) -ChildPath TestImportCsvQuoteDelimiter_QuoteCommaDelimiter.csv
+    }   
+
+    It "Should handle <name>" -TestCases @(
+		@{ name = "quote with empty value"; expectedHeader = "a1,H1,a3"; file = $TestImportCsvQuoteDelimiter_EmptyValue; delimiter = '"' }
```

Comment:
This formatting seems off, lines 33-38 have one extra indentation.  Also, it's nice to align the semicolons in the hashtable per row to make them easier to read

Reply:
Turns out those lines had tabs as indentation, instead of spaces like the rest of the lines.